packages/ui/src/lib/screen/FilteredEmptyScreen.svelte

```diff
@@ -4,19 +4,22 @@ import { createEventDispatcher } from 'svelte';
 import Button from '../button/Button.svelte';
 import EmptyScreen from './EmptyScreen.svelte';
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export let icon: any;
-export let kind: string;
-export let searchTerm: string;
-
 const dispatch = createEventDispatcher();
 
 const defaultOnResetFilter = (): void => {
   if (dispatch('resetFilter', searchTerm, { cancelable: true })) {
     searchTerm = '';
   }
 };
-export let onResetFilter: () => void = defaultOnResetFilter;
+interface Props {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
use proper typing instead of any, I'm pretty sure we have other UI components with icon not being anyas it's basically
```
        {#if fontAwesomeIcon}
          <Icon icon={icon} size='4x'/>
        {:else}
          <Icon icon={icon} size=55/>
        {/if}
```

and if I look at Icon component I see

```
interface Props {
  icon: IconDefinition | Component | string;
```

so should use the same definition